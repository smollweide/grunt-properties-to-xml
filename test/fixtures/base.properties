# base i18n properties
item1 = Item 1
item2 = Item 2
item3 = Item 3 = something