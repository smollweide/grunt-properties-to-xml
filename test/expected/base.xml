<label key="item1">Item 1</label>
<label key="item2">Item 2</label>
<label key="item3">Item 3 = something</label>